WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameViewController.swift

```diff
@@ -6,6 +6,18 @@ class SiteNameViewController: UIViewController {
 
     private let creator: SiteCreator
 
+    var nameView: SiteNameView {
+        return view as! SiteNameView
```

Comment:
We can replace this with a cast in `viewWillAppear`, though I still feel this is safe. 😄